src/network/networkd-network.c

```diff
@@ -458,6 +458,7 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 .multicast_router = _MULTICAST_ROUTER_INVALID,
                 .bridge_locked = -1,
                 .bridge_mac_authentication_bypass = -1,
+                .vlan_tunnel = -1,
```

Comment:
Please prefix with `bridge_` for new element.

Reply:
Done